connector/src/main/java/org/jboss/as/connector/logging/ConnectorLogger.java

```diff
@@ -778,13 +778,13 @@
     DeploymentUnitProcessingException invalidConnectionFactory(String cf, String ra, String jndiName);
 
     /**
-     * Admin object declared for JCA 1.0 archive
+     * Admin object declared for Jakarta Connectors archive
```

Comment:
I'm unsure about this change, this is a feature of a specific version of JCA spec, I think we should keep it as is